Frontend/src/app/components/landing-page/landing-page.component.ts

```diff
@@ -0,0 +1,74 @@
+import { Component, OnInit } from '@angular/core';
+import { Router } from '@angular/router';
+
+@Component({
+  selector: 'app-landing-page',
+  templateUrl: './landing-page.component.html',
+  styleUrls: ['./landing-page.component.css'],
+})
+export class LandingPageComponent implements OnInit {
+  constructor(private router:Router) {}
+
+  loginModel = {
+    email:'',
+    password:''
+  }
+
+  ngOnInit(): void {
+    const createBtn = document.getElementById('create-new-account-btn');
+    const loginMenu = document.getElementById('login-menu');
```

Comment:
I think instead of doing all this manually, we can use angular event binding, What you say @Himansh1706  ?